diff --git a/apps/common/CMakeLists.txt b/apps/common/CMakeLists.txt
index a621054b..af183578 100644
--- a/apps/common/CMakeLists.txt
+++ b/apps/common/CMakeLists.txt
@@ -9,4 +9,4 @@ set(src
   )
 
 add_library(apps_common ${src})
-target_link_libraries(apps_common nabto_device_static 3rdparty_cjson 3rdparty_mbedtls nn)
+target_link_libraries(apps_common nabto_device_static 3rdparty_cjson 3rdparty_mbedtls nn m)
